scripts/fuzz/generate_decoding_fuzz_corpus.py

```
@@ -0,0 +1,177 @@
+        ) from error
+    except subprocess.CalledProcessError as error:
+        raise SystemExit(
+            f"command failed with exit code {error.returncode}: {' '.join(args)}"
+        ) from error
+
+
+def write_json(path: pathlib.Path, value: object) -> None:
+    serialized = json.dumps(value, sort_keys=True, separators=(",", ":")) + "\n"
+    path.write_text(serialized)
+    if path.stat().st_size == 0:
+        raise SystemExit(f"generated empty fuzz corpus file: {path}")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)